BlueMapCommon/webapp/public/map_data_provider.php

```diff
@@ -111,9 +110,12 @@ function getMimeType($path) {
     $mapId = $pathParts[0];
     $mapPath = explode("?", $pathParts[1], 2)[0];
 
-    // get sql-connection
-    $sql = new mysqli($hostname, $username, $password, $database, $port);
-    if ($sql->errno) error(500, "Failed to connect to Database!");
+    // Initialize PDO
+    try {
+        $sql = new PDO("$driver:host=$hostname;dbname=$database", $username, $password);
+        $sql->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+    } catch (PDOException $e ) { error(500, "$e"); }
```

Comment:
Which error-messages are being outputted here, and can we be sure that this is nothing which could be dangerous to send to the public? (E.g. backend-ip's/adresses, users etc..)